### PR TITLE
Adding loading state to calendar filters

### PR DESCRIPTION
**Pull request by noahmanger**
Adds the loading / success states to calendar filters.

Requires https://github.com/18F/fec-style/pull/435

### CODE REVIEW

File: fec/fec/static/js/calendar.js [hunk 1]
```diff
@@ -146,6 +146,15 @@ Calendar.prototype.filter = function() {
 
 Calendar.prototype.success = function(response) {
   var self = this;
+
+  setTimeout(function() {
+    $('.is-loading').removeClass('is-loading').addClass('is-successful');
+  }, 500);
+
+  setTimeout(function() {
```
Comment (xtine) [inline fec/fec/static/js/calendar.js]: At first I feel kind of iffy that the second `setTimeout` relies on the first one to finish, but not sure if nesting setTimeouts is that much better. However I think setTimeouts are ever stopped without explicitly doing so.

Comment (noahmanger) [inline fec/fec/static/js/calendar.js]: What would you recommend?

**Comment by noahmanger** [comment]
Just noting that this will also need to be merged into the release branch: https://github.com/18F/fec-cms/pull/421

**Comment by ccostino** [comment]
@noahmanger and @xtine:  Just double checking that this is ready to go - it looks fine to me (and noted about the `setTimeout` chaining, but I've had to do similar things in the past), but just wanted to confirm.

File: fec/fec/static/js/calendar.js (hunk 1 above)
Comment (xtine) [inline fec/fec/static/js/calendar.js]: It was just a general note, I don't think I have anything better to suggest. The rest looks good to me! 👍

**Comment by xtine** [comment]
👍

**Pull request merged**